benefits/core/templates/core/help.html

```diff
@@ -3,48 +3,47 @@
-<p>{% translate "core.pages.help.login_gov.p[1]" %}</p>
-<p>{% blocktranslate with website="https://login.gov"%}core.pages.help.login_gov.p[2]{{ website }}{% endblocktranslate %}</p>
-
-<h2 id="littlepay">{% translate "core.pages.help.littlepay" %}</h2>
-<p>{% translate "core.pages.help.littlepay.p[0]" %}</p>
-<p>{% blocktranslate with website="https://littlepay.com" %}core.pages.help.littlepay.p[1]{{ website }}{% endblocktranslate %}</p>
-
-<h2 id="questions">{% translate "core.pages.help.questions" %}</h2>
-<p>{% translate "core.pages.help.questions.p[0]" %}</p>
-
-{% block buttons %}
-{{ block.super }}
-{% endblock %}
-
-<p class="mt-4">
-    {% translate "core.pages.help.foss.text" %}
-    <a href="https://www.github.com/cal-itp/benefits" target="_blank" rel="noopener noreferrer">{% translate "core.pages.help.foss.link" %}</a>.
-</p>
-{% endblock %}
+    <div class="container-fluid">
+        <div class="container content help">
+            {% block container_content %}
+                <h1>{{ page.content_title }}</h1>
+
+                <h2 id="what-is-cal-itp">{% translate "core.pages.help.about" %}</h2>
+                <p>{% translate "core.pages.help.about.p[0]" %}</p>
+                <p>{% translate "core.pages.help.about.p[1]" %}</p>
+
+                <h2 id="payment-options">{% translate "core.pages.help.payment_options" %}</h2>
+                <p>{% translate "core.pages.help.payment_options.p[0]" %}</p>
+
+                <p class="contactless-symbol">
+                    <img class="icon mx-auto d-block" src="{% static 'img/icon/contactless.svg' %}" alt="{% translate "core.icons.contactless" context "image alt text" %}" />
+                </p>
+
+                <p>{% translate "core.pages.help.payment_options.p[1]" %}</p>
+                <p>{% translate "core.pages.help.payment_options.p[2]" %}</p>
+                <p>{% translate "core.pages.help.payment_options.p[3]" %}</p>
+
+                <h2 id="login-gov">{% translate "core.pages.help.login_gov" %}</h2>
+                <p>{% translate "core.pages.help.login_gov.p[0]" %}</p>
+                <p>{% translate "core.pages.help.login_gov.p[1]" %}</p>
+                <p>{% blocktranslate with website="https://login.gov"%}core.pages.help.login_gov.p[2]{{ website }}{% endblocktranslate %}</p>
+
+                <h2 id="littlepay">{% translate "core.pages.help.littlepay" %}</h2>
+                <p>{% translate "core.pages.help.littlepay.p[0]" %}</p>
+                <p>{% blocktranslate with website="https://littlepay.com" %}core.pages.help.littlepay.p[1]{{ website }}{% endblocktranslate %}</p>
+
+                <h2 id="questions">{% translate "core.pages.help.questions" %}</h2>
+                <p>{% translate "core.pages.help.questions.p[0]" %}</p>
+
+                {% block buttons %}
+                    {% include "core/includes/help-links.html" with buttons=page.buttons %}
+                {% endblock buttons %}
```

Comment:
I realize that I wrote the ticket geared towards the Help page, so let me know if you think this is in or out of scope for this PR...

The agency links are also shown on some error pages, and the Figma mockup shows them as one line with just a phone number as well: https://www.figma.com/file/5IabSdY0xDzwL4gyXhbQ9G/Benefits-(Login.gov)?node-id=2083%3A8058

![image](https://user-images.githubusercontent.com/25497886/165183285-af89597a-79ce-409f-aaa7-c48a20031eef.png)

Would we just need to move this change to `page.html` instead? And rename the includes to `links.html`?

Reply:
Thank you for this reminder @angela-tran!

I will test this page too and make any changes necessary.

Reply:
Upcoming --> <img width="1512" alt="image" src="https://user-images.githubusercontent.com/3673236/165198940-c596f685-643e-4279-b869-fd6e2ca82b20.png">